CHANGELOG.md

```diff
@@ -3,10 +3,12 @@
 ### Features
 - #58 Add a defined API for every field type, and ensure all fields implement it (Thanks @alexander-zw)
 - #71 Add BLS12-381 (Thanks @yelhousni)
+- #79 Separate field initialization from curves (Thanks @alexander-zw)
```

Comment:
Lets also add upgrade notes for how to change old code for the new PR! I can help work on this as well, this can definitely be done in a separate PR, just needs to be done before the next release.